searx/search/processors/online.py

```diff
@@ -29,7 +30,6 @@ def default_request_params():
         'data': {},
         'url': '',
         'cookies': {},
-        'verify': True,
```

Comment:
Don't override the value provide by the network = the value defined in `settings["outgoing"]["verify"]`